recipes/marisa-trie/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{%set name = "marisa-trie" %}
+{%set version = "0.7.2" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "4b39f84c511a6e8e3a060b20bc41ff3b5393cb46381336e2156dc9b2290506b3" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+  patches:
+    - louds-trie.patch  # [win and py35]
+    # patch for windows build issue; see 18831 in original repo
```

Comment:
It's been submitted upstream on github (see [here](https://github.com/pytries/marisa-trie/commit/1883107b420e19b30f18b1f92b19d7f9d536fa5d)) but isn't in the latest build on pypi.